web-app/js/portal/ui/openlayers/layer/NcWms.js

```diff
@@ -60,7 +60,9 @@ OpenLayers.Layer.NcWms = OpenLayers.Class(OpenLayers.Layer.WMS, {
                     this._parseDatesWithDataAsync(Ext.util.JSON.decode(resp.responseText));
                 }
                 catch (e) {
-                    log.error("Could not parse filters for NcWMS layer '" + this.params.LAYERS + "'");
+                    log.error("Could not parse dates for NcWMS layer '" + this.params.LAYERS + "'");
+                    // Dataset is missing date information aodn/backlog#594
+                    this.events.triggerEvent('temporalextentloaded', this);
```

Comment:
Should we just check to see if there's any date info rather than triggering this on an exception (possibly unrelated to date info missing)